### PR TITLE
Some changes in Tests

### PR DESCRIPTION
**Pull request by wenqing**
* Set createTestProperty as a common function for MPL test.
* Added namespace to two common functions.
* Fixed the error that new file in Tests gives a conflict with VTK headers.

### CODE REVIEW

**Comment by endJunction** [comment]
clang-format should be executed on each commit.

**Comment by bilke** [comment]
Does this fix the unity build issue you described yesterday in Mattermost?

**Comment by wenqing** [comment]
> Does this fix the unity build issue you described yesterday in Mattermost?

Not yet. I am trying to find the reason. If I fix it, I will add the fixing to this PR.

File: Tests/MaterialLib/TestMPL.h [hunk 1]
```diff
@@ -13,9 +13,33 @@
 
 #pragma once
 
-#include "Tests/TestTools.h"
+#include <functional>
+#include <memory>
 
+#include "Tests/TestTools.h"
```
[the remark this answers is not in the document]
Comment (wenqing) [reply Tests/MaterialLib/TestMPL.h]: Removed.

**Comment by wenqing** [comment]
> clang-format should be executed on each commit.

Ran git-clang-format but it seems take no effect.

**Comment by wenqing** [comment]
> Does this fix the unity build issue you described yesterday in Mattermost?

The problem
```
/usr/include/vtk/vtkAtomic.h:360:11: error: reference to ‘detail’ is ambiguous

```
 is caused by the namespace name of `detail`, which is used in VTK as well. The last commit of this PR fixed the issue.

File: Tests/TestTools.cpp [hunk 2]
```diff
@@ -11,14 +11,16 @@
...
+boost::property_tree::ptree readXml(const char xml[])
 {
     boost::property_tree::ptree ptree;
     std::istringstream xml_str(xml);
     boost::property_tree::read_xml(
-        xml_str, ptree,
-        boost::property_tree::xml_parser::no_comments |
-        boost::property_tree::xml_parser::trim_whitespace);
+        xml_str, ptree, boost::property_tree::xml_parser::no_comments |
+                            boost::property_tree::xml_parser::trim_whitespace);
```
Comment (endJunction) [inline Tests/TestTools.cpp:22]: this formatting is still rather random. Which clang version do you use?

**Comment by endJunction** [comment]
Which vtk version?

File: Tests/TestTools.cpp (hunk 2 above)
Comment (wenqing) [reply Tests/TestTools.cpp:22]: clang-format version 10.0.0

**Comment by wenqing** [comment]
> Which vtk version?

vtk-8.2.0-13

File: Tests/TestTools.cpp (hunk 2 above)
Comment (wenqing) [reply Tests/TestTools.cpp:22]: Ran several times, but that ugly format does not change.

**Comment by wenqing** [comment]
Any further comments on this PR?

File: Tests/TestTools.cpp (hunk 2 above)
Comment (endJunction) [reply Tests/TestTools.cpp:22]: Should be fixed. Actually there is no change in those lines, remove them from the PR.

Comment (wenqing) [reply Tests/TestTools.cpp]: Recovered to the previous format.

**Pull request merged**